.github/workflows/compatibility.yml

```diff
@@ -40,7 +40,7 @@ jobs:
       # run: if [[ "${{ matrix.php }}" == '7.0' ]]; then wget https://phar.phpunit.de/phpunit-6.5.14.phar && mv phpunit-6.5.14.phar phpunit.phar; fi;
       # run CI checks
       - run: bash bin/run-ci-tests.bash
-
+  
```

Comment:
Nitpick - extra new line seems unrelated to changes.

Reply:
That's correct. The line was already in place, but it seems like my editor decided to add an extra space for some reason, which is now removed.